### ISSUE TITLE
New instance of Xournal++ gnome icon created when moving window to different monitor

### ISSUE THREAD

**Issue by triptronics** [xournalpp/xournalpp#1454]
(Please complete the following information, and then delete this line)

**Affects versions :**
 - OS: Ubuntu 19.04
 - 1.0.13

**Describe the bug**
The Ubuntu Gnome dock shows two instances of Xournal ++ when you move the Xournal ++ window to a different monitor.

**To Reproduce**
1. Have a dual monitor setup.
2. Favorite Xournal++ to the Ubuntu dock.
3. Open an instance of Xournal++
4. Drag the instance of Xournal++ to a different monitor.

**Expected behavior**
There should be only one Xournal++ icon in the Gnome dock with one orange dot. If I open another Xournal++ window, then I should see two dots and so on.

**Screenshots of Problem**
(Optional) If applicable, add screenshots to help explain your problem.
![Screenshot from 2019-09-04 21-36-55](https://user-images.githubusercontent.com/44954029/64305413-5d8de800-cf5d-11e9-849b-7a84bd1ddd72.png)

**Additional context**
I don't recall this being a problem before, this just started happening maybe a few updates ago? Not sure.

**Comment by triptronics** [comment]
Alright, after some digging arround I think I have found a temporary workaround for this issue. Following the instructions in this video here: https://www.youtube.com/watch?v=xZ_M3Q-U-J0 

I added the StartupWMClass=xournalpp to the desktop file and then saved the file and tried again. Now everything is working as expected. No matter how many xournalpp windows I have open or what monitor they are on, there is just one xournalpp icon with multiple orange dots.

 I say that this is a temporary fix because I checked the desktop icons for other programs like VLC and Zim etc.. and they do not have this StartupWMClass in their .desktop file yet they still work as expected? 

This problem seems to be Xournalpp specific.

Looking forward to hearing back, thanks!
